dockerfiles/run/guice/cassandra-ldap/destination/conf/batchsizes.properties

```diff
@@ -0,0 +1,4 @@
+metadata=200
```

Comment:
There is also two operations available on StoreMailboxManager:

moveBatchSize and copyBatchSize. Maybe we can set it from here at the same time :-)